core/collateral/engine.go

```diff
@@ -3026,8 +3026,15 @@ func (e *Engine) ClearInsurancepool(ctx context.Context, mktID, asset string, cl
 	}
 
 	// get all other market insurance accounts for the same asset
+	sortedAccKeys := make([]string, 0, len(e.accs))
+	for k := range e.accs {
+		sortedAccKeys = append(sortedAccKeys, k)
+	}
```

Comment:
Not a big deal at all, but could be written like so I guess

```suggestion
	sortedAccKeys := maps.Keys(e.accs)
```

Reply:
Well this is going to need the import as well.